dll/win32/comctl32/listbox.c

```diff
@@ -1129,7 +1129,9 @@ static LRESULT LISTBOX_Paint( LB_DESCR *descr, HDC hdc )
         /* keep the focus rect, to paint the focus item after */
         if (i == descr->focus_item)
             focusRect = rect;
```

Comment:
Do we need the `focusRect` to have its bottom set to the "original" `rect.bottom` calculated above, or to the "corrected" (i.e. your fix) `rect.bottom` from below?

Reply:
@HBelusca, I tried this both ways. For your request, I moved the inserted code above the ```if (i == descr->focus_item)``` line. When I tested this by selecting a line within the listbox and scrolling down, I could not detect any perceptible differences either way. If you want me to move the code above this, I will be happy to do so. Thanks.

Reply:
@Doug-Lyons Could you make a screenshot similar to the "Item 16" screenshot in the "Before" case, where the Item 16 line is selected and focused? I want to see how the selection (+ the blue focus) rectangle is displayed.
And also, another screenshot of the same thing but with the current PR fix included.

Reply:
@HBelusca, Requested screen shots added at the bottom of the description. Thanks.

Reply:
Thanks Doug. While I'm surprised the usual "blue" selection doesn't show (maybe it's because the app disabled this style for the listbox), we can observe that when the focus rectangle is drawn, part (but not all) of the lower edge of the listbox control has been redrawn: see the partly-drawn grey horizontal line. (This line fully shows up in the "After-fix" version.)

<img width="538" height="142" alt="image" src="https://github.com/user-attachments/assets/b292f7ea-99b3-4ab4-ae58-acc7d444ca59" />

Reply:
@HBelusca, I have now included a W2K3SP2 screen shot for comparison after Line 16 has been clicked and the focus it on it. You can see that the box interior has been automatically scrolled upwards and now we see part of Line 17. Thanks.